### PR TITLE
Adding code coverage validation to Danger

### PR DESCRIPTION
**Pull request by BrunoMazzo**
I builded a Danger plugin to validate xcode code coverage on a PR using Slather. Adding it, Danger will start notify as warning files that have less than 50% code coverage and are changed and warning builds with less than 80%.

When we achieve this coverage we can change for fail the build in a PR.

It's marked as WIP because I'm still working on the plugin. If you wanna take a look on it, please be welcome. [Plugin Danger-Slather](https://github.com/BrunoMazzo/Danger-Slather)

### CODE REVIEW

**Comment by BrunoMazzo** [comment]
Can I merge it?
After it, I have more two PRs of tests to push. =D

**Comment by diogot** [comment]
I'll see what you added since last review

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 
+rvm:
+  - 2.3.1
+
+cache:
+  directories:
+    - bundle
+
 install:
+  # - bundle install
```
Comment (diogot) [inline .travis.yml]: Is this necessary?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,15 @@
 language: objective-c
 osx_image: xcode8
 
+rvm:
+  - 2.3.1
+
+cache:
+  directories:
+    - bundle
```
Comment (diogot) [inline .travis.yml]: Good! Worth do the same for Pods?

File: MyWeight/Screens/List/ListViewController.swift [hunk 3]
```diff
@@ -39,6 +39,7 @@ public class ListViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
+
```
Comment (diogot) [inline MyWeight/Screens/List/ListViewController.swift]: Is this intentional?

**Comment by diogot** [comment]
Ah, one thing that I just notice, you change travis configuration. Travis is disabled, I'm using only CircleCI.

**Comment by BrunoMazzo** [comment]
I will close this, organize the commits and redo it. I was testing on a travis at my fork. The line added was to test the plugin.

**Comment by diogot** [comment]
Are you sure?

**Pull request closed without merging**